rails_event_store_active_record/lib/rails_event_store_active_record/mappers/yaml.rb

```diff
@@ -0,0 +1,31 @@
+module RailsEventStoreActiveRecord
+  module Mappers
+    class YAML
+      def initialize(serializer: nil)
+        @serializer = serializer || ::YAML
+      end
```

Comment:
I keep DI here, to separate testing of serialiser and mapper (with YAML we are making assumption that its well tested, for mapper we will verify if he "mapping" it correctly with using serialiser)

Reply:
I don't think that's necessary. A YAML mapper should use YAML internally, I see no benefit in passing `YAML` as dependency.

Reply:
👍 , I will remove it